__tests__/package.json

```diff
@@ -1,28 +1,22 @@
 {
   "name": "eslint-config-upleveled-tests",
   "version": "1.0.0",
-  "main": "index.js",
-  "author": "Karl Horky <karl.horky@gmail.com>",
-  "license": "MIT",
   "private": true,
+  "license": "MIT",
+  "author": "Karl Horky <karl.horky@gmail.com>",
+  "main": "index.js",
   "scripts": {
     "update-config-version": "pnpm add --save-dev eslint-config-upleveled@latest && pnpm upleveled-eslint-install"
   },
   "dependencies": {
     "next": "13.3.0"
   },
   "devDependencies": {
-    "@ts-safeql/eslint-plugin": "^0.0.28",
-    "eslint-config-upleveled": "^4.0.12",
-    "libpg-query": "^15.0.2",
-    "postcss-styled-syntax": "^0.4.0",
-    "stylelint": "^15.4.0",
-    "stylelint-config-css-modules": "^4.2.0",
-    "stylelint-config-recommended": "^11.0.0",
-    "stylelint-config-recommended-scss": "^10.0.0",
+    "@ts-safeql/eslint-plugin": "0.0.28",
+    "eslint-config-upleveled": "4.0.13",
+    "libpg-query": "15.0.2",
+    "postcss-styled-syntax": "0.4.0",
+    "stylelint-config-upleveled": "1.0.2",
```

Comment:
Add dependencies that should be installed with pnpm add --save-dev eslint-config-upleveled@latest that are necessary to run the tests inside of this directory

Reply:
https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/2bf8840f52e0b13dd127f65d00fc147a4c5c5452
https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/690a06c9694734491dca25dcd2d22f6b39f76e45